rust/tests/read_delta_test.rs

```diff
@@ -41,6 +41,7 @@ async fn read_delta_table_with_update() {
     let table_newest_version = deltalake::open_table(path).await.unwrap();
     let mut table_to_update = deltalake::open_table_with_version(path, 0).await.unwrap();
     table_to_update.update().await.unwrap();
+    table_to_update.update().await.unwrap();
```

Comment:
could we have a dedicated test case for checking data duplication on repeated updates? that way, future developers won't accidentally remove this extra update call here thinking it's just a typo.

Reply:
done

Reply:
oh, meaning I've added a comment on top of it. But I'd like to also have another test